src/course-outline/section-card/SectionCard.jsx

```diff
@@ -174,15 +176,21 @@ const SectionCard = ({
             />
           )}
           <div className="section-card__content" data-testid="section-card__content">
-            {explanatoryMessage && <p className="text-secondary-400 x-small mb-1">{explanatoryMessage}</p>}
+            <XBlockStatus
+              isSelfPaced={isSelfPaced}
+              isCustomRelativeDatesActive={isCustomRelativeDatesActive}
+              blockData={section}
+            />
```

Comment:
Move this below the section highlights

Reply:
It is above section highlights in legacy, do we want to change it here?

Reply:
Yes

Reply:
Done.